ntbs-service/DataAccess/DataQualityRepository.cs

```diff
@@ -102,6 +104,39 @@ public async Task<IList<Notification>> GetNotificationsEligibleForDotVotAlerts()
                 .ToListAsync();
         }
 
+        public async Task<IList<NotificationAndDuplicateIds>> GetNotificationIdsEligibleForPotentialDuplicateAlerts()
+        {
+            return await GetBaseNotificationQueryableForAlerts()
```

Comment:
this expression is a bit messy given it's nature, I've left it here instead of an expression on DataQualityPotentialDuplicateAlert, any preference on where it lives?

Reply:
The main reason I've moved others to their models is to enable code-sharing with the auto-close functionality, so there's no *need* to move this one.
Given it can't be applied in a similar way (as it needs to operate on two notifications), we wouldn't be gaining that much on consistency, so perhaps it is best to keep it here 🤔